source/MBF.SAMD51.GPIO.pas

```diff
@@ -98,6 +121,69 @@         TArduinoPin = record
         A4 = TNativePin.PB8;   A5 = TNativePin.PB9;
       end;
     {$endif}
+    {$ifdef fpc_mcu_wio_terminal}
+      type
+        TArduinoPin = record
+      const
+        None=-1;
+        D13= TNativePin.PA15;
+      end;
+    {$endif}
+
+  {$endif}
+
+  {$ifdef fpc_mcu_wio_terminal}
+  const
+    // LEDs
+    PIN_LED_13	                 = 13;
```

Comment:
You are defining const here that will get a global scope and by this break the complete naming scheme resulting in no help in Lazarus on parameters.

define wiopins and put the consts below the wiopins.

Reply:
These are needed for the fatfreertos functions and to easily translate the Wio Terminal c examples.
All available FPC examples are direct copies of available c examples.
The c examples use these definitions. So, all Wio examples can very easily be understood and translated into FPC examples.